design-system-docs/frontend/styles/_base.scss

```diff
@@ -7,17 +7,6 @@ body {
   height: 100%;
 }
 
-// https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/
```

Comment:
was this causing problems with the navigation?

Reply:
Yes as a side-effect of porting over the overflow fix from public-website rather than the refactor itself.